charts/kube-prometheus-stack/values.yaml

```diff
@@ -3303,6 +3303,42 @@ prometheus:
       #   hosts:
       #     - prometheus.example.com
 
+  # -- BETA: Configure the gateway routes for the chart here.
+  # More routes can be added by adding a dictionary key like the 'main' route.
+  # Be aware that this is an early beta of this feature,
+  # kube-prometheus-stack does not guarantee this works and is subject to change.
+  # Being BETA this can/will change in the future without notice, do not use unless you want to take that risk
```

Comment:
@QuentinBisson what did you think about that? We I understand this as long as this is BETA, we are allowed todo breaking changes without bump the major.

WDYT?

Reply:
Looks good :)